lmfit/printfuncs.py

```diff
@@ -278,29 +278,29 @@ def cell(x, cat='td'):
         cell(h, cat='th')
     add('</tr>')
 
-    for p in params.values():
-        rows = [p.name, gformat(p.value)]
+    for par in params.values():
+        rows = [par.name, gformat(par.value)]
         if has_err:
-            serr, perr = '', ''
-            if p.stderr is not None:
-                serr = gformat(p.stderr)
+            serr = ''
+            if par.stderr is not None:
+                serr = gformat(par.stderr)
             try:
```

Comment:
shouldn't the indentation be at the same position as the line above, similar to what's done in the ```fit_report``` function? (https://github.com/lmfit/lmfit-py/blob/e465b8269aaa447d10b60e8139cc24ddb1259757/lmfit/printfuncs.py#L174-L179)

Reply:
yes, you are right... will fix.